.travis.yml

```diff
@@ -12,7 +12,7 @@ env:
   - PUPPET_VERSION="~> 3.4.0"
   - PUPPET_VERSION="~> 3.6.0"
   - PUPPET_VERSION="~> 3.7.0"
-  - PUPPET_VERSION="~> 4.3.0"
+  - PUPPET_VERSION="~> 4.3.0" STRICT_VARIABLES=yes
```

Comment:
we should have builds with ``STRICT_VARIABLES=yes`` and ``STRICT_VARIABLES=no`` for ``puppet=4.x``